test/test_jit.py

```diff
@@ -10189,6 +10189,17 @@ def torch_unique(dim: Optional[int]):
         self.checkScript(torch_unique, (None,))
         self.checkScript(torch_unique, (0,))
 
+        def torch_unique_consecutive(dim: Optional[int]):
+            ten = torch.unique(torch.tensor([[1, 3], [2, 3]], dtype=torch.long))
```

Comment:
maybe add some consecutive values that are the same

Reply:
@eellison That is a good idea! I have added consecutive values for the test.

Could you land this for me? I don't work for FB, so I can not land it by myself.